hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/schema/DataConnectionResolver.java

```diff
@@ -113,6 +113,13 @@ public void removeDataConnection(String name, boolean ifExists) {
         }
     }
 
+    /**
+     * Invoke all registerer change listeners
+     */
+    public void invokeChangeListeners() {
```

Comment:
maybe add TODO here as well, this method should not be public

Reply:
334545ba42f